spinalcordtoolbox/scripts/sct_deepseg_sc.py

```diff
@@ -94,6 +94,10 @@ def get_parser():
         help='Output folder. Example: My_Output_Folder/ ',
         action=ActionCreateFolder,
         default=os.getcwd())
+    optional.add_argument(
+        "-o",
```

Comment:
All the other options use single quotes

```suggestion
        '-o',
```